x/ccv/provider/keeper/relay.go

```diff
@@ -34,14 +34,24 @@ func (k Keeper) OnRecvVSCMaturedPacket(
 	packet channeltypes.Packet,
 	data ccv.VSCMaturedPacketData,
 ) exported.Acknowledgement {
-	// check that the channel is established
+
+	// check that the channel is established, panic if not
 	chainID, found := k.GetChannelToChain(ctx, packet.DestinationChannel)
 	if !found {
 		// VSCMatured packet was sent on a channel different than any of the established CCV channels;
 		// this should never happen
 		panic(fmt.Errorf("VSCMaturedPacket received on unknown channel %s", packet.DestinationChannel))
 	}
 
+	// Note: no validation is needed (that'd return an IBC err ack) for recv VSCMatured packets,
+	// since the packet data only includes a valset update id that can take any uint64 value.
```

Comment:
Actually it cannot be `0`, but probably it's not worth checking this case. 

More relevant would be to check whether subsequent `VSCMaturedPacket`s have increasing `vscID`s. Otherwise, a malicious consumer could send a lot of `VSCMaturedPacket`s  with random `vscID`s (or `0`), which means that relayers will end up relaying garbage without even realizing it (no err ack). 

We could actually check if every `VSCMaturedPacket` has the next expected `vscID` (i.e., the next `vscID` in `UnbondingOpsFromIndex`).

Reply:
> We could actually check if every VSCMaturedPacket has the next expected vscID (i.e., the next vscID in UnbondingOpsFromIndex).

This is actually incorrect since some `VSCPackets` can be sent without any unbonding operations.

Reply:
This point will be superseeded by #568, I've added a comment in that issue specifically about validating VSCMatured packets